Sources/NIO/EventLoopFuture.swift

```diff
@@ -828,14 +828,41 @@ extension EventLoopFuture {
     ///     - eventLoop: The `EventLoop` on which the new `EventLoopFuture` callbacks will fire.
     /// - returns: A new `EventLoopFuture`.
     public static func andAll(_ futures: [EventLoopFuture<Void>], eventLoop: EventLoop) -> EventLoopFuture<Void> {
-        let p0: EventLoopPromise<Void> = eventLoop.newPromise()
+        let body = EventLoopFuture<Void>.reduce((), futures, eventLoop: eventLoop) {
+            (_, _) in
```

Comment:
Can we add the types that we're explicitly suppressing here? We've found that it reduces the number of bugs we hit by doing so. That'd be `(_: Void, _: Void) in`.